concepts/05 Widgets/Calendar/20 Keyboard Support.md

```diff
@@ -73,6 +73,58 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxCalendar @initialized="registerKeyHandlers" />
```

Comment:
We usually call registerKeyHandler on `mounted`/`componentDidMount()`:
https://js.devexpress.com/Documentation/20_1/Guide/Widgets/Button/Keyboard_Support/